tar/chunks/chunks_stats.c

```diff
@@ -414,3 +414,45 @@ chunks_stats_free(CHUNKS_S * C)
 	free(C->cachepath);
 	free(C);
 }
+
+/**
+ * chunks_initialize(const char * cachepath):
+ * Initialize the chunk directory (file) in the ${cachepath}.
+ */
+int
+chunks_initialize(const char * cachepath)
+{
+	struct chunks_stats_internal * C;
```

Comment:
We're not returning a cookie to be passed back to us later, so we don't need this.